pkg/cmd/experimental/syncgroups/groupsyncer.go

```diff
@@ -63,7 +63,7 @@ func (s *LDAPGroupSyncer) Sync() ([]*userapi.Group, []error) {
 		// get membership data
 		memberEntries, err := s.GroupMemberExtractor.ExtractMembers(ldapGroupUID)
 		if err != nil {
-			fmt.Fprintf(s.Err, "Error determining LDAP group membership for %q: %v.  Continuing with the next LDAP group.", ldapGroupUID, err)
+			fmt.Fprintf(s.Err, "Error determining LDAP group membership for %q: %v.  Continuing with the next LDAP group.\n", ldapGroupUID, err)
```

Comment:
kill ".  Continuing with the next LDAP group." globally, I think... there may not be a next group... you'll see status on the next group when it prints